### ISSUE TITLE
RStudio drops part query text when using str_glue and Google BigQuery

### ISSUE THREAD

**Issue by yonghaoy** [rstudio/rstudio#14420]
### System details

    RStudio Edition : 
    RStudio Version : 4.1.0
    OS Version      : linux
    R Version       : 4.2

Actually we are using https://github.com/Bioconductor/bioconductor_docker/tree/devel 3.18
### Describe the problem in detail
```
library(tidyverse)
library(bigrquery)
download_data <- function(query) {
    tb <- bq_project_query(Sys.getenv('GOOGLE_PROJECT'), query = query, page_size = 50000
                           , default_dataset = Sys.getenv('WORKSPACE_CDR'))
    bq_table_download(tb, bigint = "integer64")
}

dataset_44665679_condition_sql <- **str_glue (extreme large query text here)**
dataset_44665679_condition_df <- download_data(dataset_44665679_condition_sql)

dim(dataset_44665679_condition_df) 

head(dataset_44665679_condition_df, 5)
n_distinct(dataset_44665679_condition_df$person_id)
```
We observe way less result than expected. Then we check BQ audit logs, turns out part of query inside a `IN` block are dropped. 

**More details about this query:** 
We have a giant query which has a large `WHERE concept_id IN ()` block, inside that IN, we pass **1421** concept_id, but RStudio only passes the **416** to BigQuery, the rest of them are missing. 

### Describe the behavior you expected
We want BQ execute the exact query we want it to execute

- [ ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ ] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.posit.co/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [ ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jmthibault79** [comment]
I'm on Yonghao's team.  We do not see this problem when executing the same query in another environment (Jupyter)

**Comment by kevinushey** [comment]
Sorry to hear you're having trouble. How are you executing the code -- via Ctrl + Enter / the Run button, or are you using `source()` to run the script?

I ask because the R parser has a limit on the number of characters that can be submitted at a time, so I wonder if the issue here could be because the query string you're constructing is so large, and it's getting truncated inside the R parser.

It would also help if you were able to share a standalone, reproducible example -- e.g. something I could run myself locally to try and reproduce.

**Comment by yonghaoy** [comment]
Thanks Kevin!
That's a giant query (but also our on of the most common use cases ) generated from another step, we copy the generated R code into RStudio then use Run button. 
I was also thinking it could be char limit issue, however it's strange that RStudio drops part of query in the middle and the query is still valid after dropping. 
It's something like this: 
```
concept_id IN (1, 2, 3, 4,..........400, 401,.......1000
)
``` 

It drops `,401~100`, would there be a limit of per line?

**Comment by yonghaoy** [comment]
Confirmed, there is a limit on max character per line, adding line

 break works. 

```
concept_id IN (1, 2, 3, 4,.
.........400, 
401,.......
1000
)
```

**Comment by kevinushey** [comment]
Thanks for confirming!

It's possible RStudio could help transparently handle this behind the scenes, by attempting to submit such code to the R interpreter in chunks that it could properly consume. However, I'd still recommend re-structuring your code to avoid large embedded string literals if possible.

**Comment by yonghaoy** [comment]
Yup! We are restructuring code now. It's big unfortunate that RStudio does not return any error just silently change input, that causes the wrong analysis result. 
We are just lucky that one user compare the same query in Jupyter vs RStudio then found this inconsistency...
(an error like "you reach to the maximum length" would be better)
I will close the ticket and restructure our code

**Comment by kevinushey** [comment]
> (an error like "you reach to the maximum length" would be better)

This is very true -- we should at the very least print a warning, rather than just silently truncating the code. Maybe we can keep this issue open to track that?

**Comment by ronblum** [comment]
@yonghaoy and @jmthibault79 Are you using RStudio Desktop or Server? (I'll check both, but curious.) And which version of RStudio—it says 4.1.0 above, but that looks like R; you can get the RStudio version with `RStudio.Version()$long_version`.

**Comment by yonghaoy** [comment]
```
RStudio.Version()$long_version.
NULL
```
But I got this in Help tab:
![Screenshot 2024-03-14 at 2 36 52 PM](https://github.com/rstudio/rstudio/assets/6351731/1b2f88d4-d203-41c3-bda9-ce060fbc4e67)

**Comment by ronblum** [comment]
Thanks! And sorry, the end period should be removed. I shouldn't have put `RStudio.Version()$long_version` at the end of sentence, ending it with a period...

**Comment by ronblum** [comment]
Verified using the example in QA notes in #14428. I changed the last few characters slightly in order to see the truncation at the end:
```
"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa1234567890"
```

First checked the original behavior in RStudio Desktop 2023.12.1.402 on MacOS 14.4 and RHEL 9. We wait for a closing double-quote because the quotation marks got truncated. 

<img width="493" alt="Screenshot 2024-03-18 at 15 15 03" src="https://github.com/rstudio/rstudio/assets/31009952/a5e9b2a2-d15e-4ed2-88be-f53dc7ac573a">

Entering the quotes will allow the command to complete, and we see the truncation.
<img width="487" alt="Screenshot 2024-03-18 at 15 15 16" src="https://github.com/rstudio/rstudio/assets/31009952/d4eb248c-d108-4cc4-8ca8-ce83715daad8">


With the first, checked in RStudio Desktop 2024.04.0-daily+603 on Red Hat 9 and Posit Workbench 2024.04.0-daily+599.pro1 on Ubuntu 20.04. We wait for a closing quote, but this time we warn that the line will be truncated.

<img width="434" alt="Screenshot 2024-03-18 at 15 26 12" src="https://github.com/rstudio/rstudio/assets/31009952/67e0b610-bee4-4f75-8c25-c73da1dc41bb">

With the quote, we see the promised truncation.

<img width="428" alt="Screenshot 2024-03-18 at 15 26 25" src="https://github.com/rstudio/rstudio/assets/31009952/68f79a3c-7b7c-4454-ae5b-554ec9faae06">

**Comment by BdR76** [comment]
I was just about to open a similar issue, with a generated R-script what has SQL code with multiple lines where the first line gets cut off.
```
select_sql_generated <-
  "  SELECT PAT_ID, ENCOUNTER_ID, FLO_FIELD_0001, FLO_FIELD_0002, FLO_FIELD_0003, FLO_FIELD_0004, FLO_FIELD_0005, FLO_FIELD_0006, FLO_FIELD_0007, FLO_FIELD_0008, FLO_FIELD_0009, FLO_FIELD_0010, FLO_FIELD_0011, FLO_FIELD_0012, FLO_FIELD_0013, FLO_FIELD_0014, FLO_FIELD_0015, FLO_FIELD_0016, FLO_FIELD_0017, FLO_FIELD_0018, FLO_FIELD_0019, FLO_FIELD_0020, FLO_FIELD_0021, FLO_FIELD_0022, FLO_FIELD_0023, FLO_FIELD_0024, FLO_FIELD_0025, FLO_FIELD_0026, FLO_FIELD_0027, FLO_FIELD_0028, FLO_FIELD_0029, FLO_FIELD_0030, FLO_FIELD_0031, FLO_FIELD_0032, FLO_FIELD_0033, FLO_FIELD_0034, FLO_FIELD_0035, FLO_FIELD_0036, FLO_FIELD_0037, FLO_FIELD_0038, FLO_FIELD_0039, FLO_FIELD_0040, FLO_FIELD_0041, FLO_FIELD_0042, FLO_FIELD_0043, FLO_FIELD_0044, FLO_FIELD_0045, FLO_FIELD_0046, FLO_FIELD_0047, FLO_FIELD_0048, FLO_FIELD_0049, FLO_FIELD_0050, FLO_FIELD_0051, FLO_FIELD_0052, FLO_FIELD_0053, FLO_FIELD_0054, FLO_FIELD_0055, FLO_FIELD_0056, FLO_FIELD_0057, FLO_FIELD_0058, FLO_FIELD_0059, FLO_FIELD_0060, FLO_FIELD_0061, FLO_FIELD_0062, FLO_FIELD_0063, FLO_FIELD_0064, FLO_FIELD_0065, FLO_FIELD_0066, FLO_FIELD_0067, FLO_FIELD_0068, FLO_FIELD_0069, FLO_FIELD_0070, FLO_FIELD_0071, FLO_FIELD_0072, FLO_FIELD_0073, FLO_FIELD_0074, FLO_FIELD_0075, FLO_FIELD_0076, FLO_FIELD_0077, FLO_FIELD_0078, FLO_FIELD_0079, FLO_FIELD_0080, FLO_FIELD_0081, FLO_FIELD_0082, FLO_FIELD_0083, FLO_FIELD_0084, FLO_FIELD_0085, FLO_FIELD_0086, FLO_FIELD_0087, FLO_FIELD_0088, FLO_FIELD_0089, FLO_FIELD_0090, FLO_FIELD_0091, FLO_FIELD_0092, FLO_FIELD_0093, FLO_FIELD_0094, FLO_FIELD_0095, FLO_FIELD_0096, FLO_FIELD_0097, FLO_FIELD_0098, FLO_FIELD_0099, FLO_FIELD_0100, FLO_FIELD_0101, FLO_FIELD_0102, FLO_FIELD_0103, FLO_FIELD_0104, FLO_FIELD_0105, FLO_FIELD_0106, FLO_FIELD_0107, FLO_FIELD_0108, FLO_FIELD_0109, FLO_FIELD_0110, FLO_FIELD_0111, FLO_FIELD_0112, FLO_FIELD_0113, FLO_FIELD_0114, FLO_FIELD_0115, FLO_FIELD_0116, FLO_FIELD_0117, FLO_FIELD_0118, FLO_FIELD_0119, FLO_FIELD_0120, FLO_FIELD_0121, FLO_FIELD_0122, FLO_FIELD_0123, FLO_FIELD_0124, FLO_FIELD_0125, FLO_FIELD_0126, FLO_FIELD_0127, FLO_FIELD_0128, FLO_FIELD_0129, FLO_FIELD_0130, FLO_FIELD_0131, FLO_FIELD_0132, FLO_FIELD_0133, FLO_FIELD_0134, FLO_FIELD_0135, FLO_FIELD_0136, FLO_FIELD_0137, FLO_FIELD_0138, FLO_FIELD_0139, FLO_FIELD_0140, FLO_FIELD_0141, FLO_FIELD_0142, FLO_FIELD_0143, FLO_FIELD_0144, FLO_FIELD_0145, FLO_FIELD_0146, FLO_FIELD_0147, FLO_FIELD_0148, FLO_FIELD_0149, FLO_FIELD_0150, FLO_FIELD_0151, FLO_FIELD_0152, FLO_FIELD_0153, FLO_FIELD_0154, FLO_FIELD_0155, FLO_FIELD_0156, FLO_FIELD_0157, FLO_FIELD_0158, FLO_FIELD_0159, FLO_FIELD_0160, FLO_FIELD_0161, FLO_FIELD_0162, FLO_FIELD_0163, FLO_FIELD_0164, FLO_FIELD_0165, FLO_FIELD_0166, FLO_FIELD_0167, FLO_FIELD_0168, FLO_FIELD_0169, FLO_FIELD_0170, FLO_FIELD_0171, FLO_FIELD_0172, FLO_FIELD_0173, FLO_FIELD_0174, FLO_FIELD_0175, FLO_FIELD_0176, FLO_FIELD_0177, FLO_FIELD_0178, FLO_FIELD_0179, FLO_FIELD_0180, FLO_FIELD_0181, FLO_FIELD_0182, FLO_FIELD_0183, FLO_FIELD_0184, FLO_FIELD_0185, FLO_FIELD_0186, FLO_FIELD_0187, FLO_FIELD_0188, FLO_FIELD_0189, FLO_FIELD_0190, FLO_FIELD_0191, FLO_FIELD_0192, FLO_FIELD_0193, FLO_FIELD_0194, FLO_FIELD_0195, FLO_FIELD_0196, FLO_FIELD_0197, FLO_FIELD_0198, FLO_FIELD_0199, FLO_FIELD_0200, FLO_FIELD_0201, FLO_FIELD_0202, FLO_FIELD_0203, FLO_FIELD_0204, FLO_FIELD_0205, FLO_FIELD_0206, FLO_FIELD_0207, FLO_FIELD_0208, FLO_FIELD_0209, FLO_FIELD_0210, FLO_FIELD_0211, FLO_FIELD_0212, FLO_FIELD_0213, FLO_FIELD_0214, FLO_FIELD_0215, FLO_FIELD_0216, FLO_FIELD_0217, FLO_FIELD_0218, FLO_FIELD_0219, FLO_FIELD_0220, FLO_FIELD_0221, FLO_FIELD_0222, FLO_FIELD_0223, FLO_FIELD_0224, FLO_FIELD_0225, FLO_FIELD_0226, FLO_FIELD_0227, FLO_FIELD_0228, FLO_FIELD_0229, FLO_FIELD_0230, FLO_FIELD_0231, FLO_FIELD_0232, FLO_FIELD_0233, FLO_FIELD_0234, FLO_FIELD_0235, FLO_FIELD_0236, FLO_FIELD_0237, FLO_FIELD_0238, FLO_FIELD_0239, FLO_FIELD_0240, FLO_FIELD_0241, FLO_FIELD_0242, FLO_FIELD_0243, FLO_FIELD_0244, FLO_FIELD_0245, FLO_FIELD_0246, FLO_FIELD_0247, FLO_FIELD_0248, FLO_FIELD_0249, FLO_FIELD_0250, FLO_FIELD_0251, FLO_FIELD_0252, FLO_FIELD_0253, FLO_FIELD_0254, FLO_FIELD_0255, FLO_FIELD_0256, FLO_FIELD_0257, FLO_FIELD_0258, FLO_FIELD_0259, FLO_FIELD_0260, FLO_FIELD_0261, FLO_FIELD_0262, FLO_FIELD_0263, FLO_FIELD_0264, FLO_FIELD_0265, FLO_FIELD_0266, FLO_FIELD_0267, FLO_FIELD_0268, FLO_FIELD_0269, FLO_FIELD_0270, FLO_FIELD_0271, FLO_FIELD_0272, FLO_FIELD_0273, FLO_FIELD_0274, FLO_FIELD_0275, FLO_FIELD_0276, FLO_FIELD_0277, FLO_FIELD_0278, FLO_FIELD_0279, FLO_FIELD_0280, FLO_FIELD_0281, FLO_FIELD_0282, FLO_FIELD_0283, FLO_FIELD_0284, FLO_FIELD_0285, FLO_FIELD_0286, FLO_FIELD_0287, FLO_FIELD_0288, FLO_FIELD_0289, FLO_FIELD_0290, FLO_FIELD_0291, FLO_FIELD_0292, FLO_FIELD_0293, FLO_FIELD_0294, FLO_FIELD_0295, FLO_FIELD_0296, FLO_FIELD_0297, FLO_FIELD_0298, FLO_FIELD_0299, FLO_FIELD_0300, FLO_FIELD_0301, FLO_FIELD_0302, FLO_FIELD_0303, FLO_FIELD_0304, FLO_FIELD_0305, FLO_FIELD_0306, FLO_FIELD_0307, FLO_FIELD_0308, FLO_FIELD_0309, FLO_FIELD_0310, FLO_FIELD_0311, FLO_FIELD_0312, FLO_FIELD_0313, FLO_FIELD_0314, FLO_FIELD_0315, FLO_FIELD_0316, FLO_FIELD_0317, FLO_FIELD_0318, FLO_FIELD_0319, FLO_FIELD_0320, FLO_FIELD_0321, FLO_FIELD_0322, FLO_FIELD_0323, FLO_FIELD_0324, FLO_FIELD_0325, FLO_FIELD_0326, FLO_FIELD_0327, FLO_FIELD_0328, FLO_FIELD_0329, FLO_FIELD_0330, FLO_FIELD_0331, FLO_FIELD_0332, FLO_FIELD_0333, FLO_FIELD_0334, FLO_FIELD_0335, FLO_FIELD_0336, FLO_FIELD_0337, FLO_FIELD_0338, FLO_FIELD_0339, FLO_FIELD_0340, FLO_FIELD_0341, FLO_FIELD_0342, FLO_FIELD_0343, FLO_FIELD_0344, FLO_FIELD_0345, FLO_FIELD_0346, FLO_FIELD_0347, FLO_FIELD_0348, FLO_FIELD_0349, FLO_FIELD_0350, FLO_FIELD_0351, FLO_FIELD_0352, FLO_FIELD_0353, FLO_FIELD_0354, FLO_FIELD_0355, FLO_FIELD_0356, FLO_FIELD_0357, FLO_FIELD_0358, FLO_FIELD_0359, FLO_FIELD_0360, FLO_FIELD_0361, FLO_FIELD_0362, FLO_FIELD_0363, FLO_FIELD_0364, FLO_FIELD_0365, FLO_FIELD_0366, FLO_FIELD_0367, FLO_FIELD_0368, FLO_FIELD_0369, FLO_FIELD_0370, FLO_FIELD_0371, FLO_FIELD_0372, FLO_FIELD_0373, FLO_FIELD_0374, FLO_FIELD_0375, FLO_FIELD_0376, FLO_FIELD_0377, FLO_FIELD_0378, FLO_FIELD_0379, FLO_FIELD_0380, FLO_FIELD_0381, FLO_FIELD_0382, FLO_FIELD_0383, FLO_FIELD_0384, FLO_FIELD_0385, FLO_FIELD_0386, FLO_FIELD_0387, FLO_FIELD_0388, FLO_FIELD_0389, FLO_FIELD_0390, FLO_FIELD_0391, FLO_FIELD_0392, FLO_FIELD_0393, FLO_FIELD_0394, FLO_FIELD_0395, FLO_FIELD_0396, FLO_FIELD_0397, FLO_FIELD_0398, FLO_FIELD_0399, FLO_FIELD_0400, FLO_FIELD_0401, FLO_FIELD_0402, FLO_FIELD_0403, FLO_FIELD_0404, FLO_FIELD_0405, FLO_FIELD_0406, FLO_FIELD_0407, FLO_FIELD_0408, FLO_FIELD_0409, FLO_FIELD_0410, FLO_FIELD_0411, FLO_FIELD_0412, FLO_FIELD_0413, FLO_FIELD_0414, FLO_FIELD_0415, FLO_FIELD_0416, FLO_FIELD_0417, FLO_FIELD_0418, FLO_FIELD_0419, FLO_FIELD_0420, FLO_FIELD_0421, FLO_FIELD_0422, FLO_FIELD_0423, FLO_FIELD_0424, FLO_FIELD_0425, FLO_FIELD_0426, FLO_FIELD_0427, FLO_FIELD_0428, FLO_FIELD_0429, FLO_FIELD_0430, FLO_FIELD_0431, FLO_FIELD_0432, FLO_FIELD_0433, FLO_FIELD_0434, FLO_FIELD_0435, FLO_FIELD_0436, FLO_FIELD_0437, FLO_FIELD_0438, FLO_FIELD_0439, FLO_FIELD_0440, FLO_FIELD_0441, FLO_FIELD_0442, FLO_FIELD_0443, FLO_FIELD_0444, FLO_FIELD_0445, FLO_FIELD_0446, FLO_FIELD_0447, FLO_FIELD_0448, FLO_FIELD_0449, FLO_FIELD_0450, FLO_FIELD_0451, FLO_FIELD_0452, FLO_FIELD_0453, FLO_FIELD_0454, FLO_FIELD_0455, FLO_FIELD_0456, FLO_FIELD_0457, FLO_FIELD_0458, FLO_FIELD_0459, FLO_FIELD_0460, FLO_FIELD_0461, FLO_FIELD_0462, FLO_FIELD_0463, FLO_FIELD_0464, FLO_FIELD_0465, FLO_FIELD_0466, FLO_FIELD_0467, FLO_FIELD_0468, FLO_FIELD_0469, FLO_FIELD_0470, FLO_FIELD_0471, FLO_FIELD_0472, FLO_FIELD_0473, FLO_FIELD_0474, FLO_FIELD_0475, FLO_FIELD_0476, FLO_FIELD_0477, FLO_FIELD_0478, FLO_FIELD_0479, FLO_FIELD_0480, FLO_FIELD_0481, FLO_FIELD_0482, FLO_FIELD_0483, FLO_FIELD_0484, FLO_FIELD_0485, FLO_FIELD_0486, FLO_FIELD_0487, FLO_FIELD_0488, FLO_FIELD_0489, FLO_FIELD_0490, FLO_FIELD_0491, FLO_FIELD_0492, FLO_FIELD_0493, FLO_FIELD_0494, FLO_FIELD_0495, FLO_FIELD_0496, FLO_FIELD_0497, FLO_FIELD_0498, FLO_FIELD_0499, FLO_FIELD_0500
  FROM MYSERVER.dbo.MYSTUDY
  ORDER BY PAT_ID"

print(select_sql_generated)
```
So `FLO_FIELD_0001` to `FLO_FIELD_0500` which resulted in
```
<...>FLO_FIELD_0249, FLO_FIELD_0250, FLO_FIELD_0251, FLO_FIELD_0252, FLO_FIELD_0253, FLO_FIELD_02
  FROM MYSERVER.dbo.MYSTUDY
  ORDER BY PAT_ID
```
but I assume this is the same issue which is addressed in #14428
